app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: post) do |form| %>
+  <% if post.errors.any? %>
```

Comment:
To show errors we have a shared partial `namederror_messages`, remove this block and try to do something like this

`  <%= render 'shared/error_messages', resource: post %>`